### DISCUSSION TITLE
Intro to Machine Learning

### DISCUSSION THREAD

**Discussion by soumyadeepdas28** [dsc-tiu/MLStudyJams2022#6] [Show and tell]
Hey There I am Soumyadeep ,I have successfully completed Intro to Machine Learning at Kaggle
![Soumyadeep Das - Intro to Machine Learning](https://user-images.githubusercontent.com/96656148/188942020-6b837d78-6d6c-4208-9504-e7182f343615.png)

**Comment by diffrxction** [comment]
Well done! 👏🏻 Keep up the good work and complete the next certification as well. 👍🏻
